Cargo.toml

```diff
@@ -205,7 +205,7 @@ use_debug = "deny"
 
 [profile.release]
 codegen-units = 1
-lto = true # attempts to perform optimizations across all crates within the dependency graph
+lto = "thin" # attempts to perform optimizations across all crates within the dependency graph
```

Comment:
comment is kind of stale now

Reply:
"thin" is still different than true where true is thin-local, i.e. thin over local crates and thin is thin over all packages